signals.py

```diff
@@ -91,14 +100,25 @@ def call(self, input, *args, **kwargs):
             else:
                 noise_weights = 1
 
-            stdd = abs(tf.math.reduce_mean(signal, axis=1)) / self._snr  # calculate standard deviation for each signal
-            stdd = tf.repeat(tf.expand_dims(stdd, 1), 11, axis=1)
+            #stdd = abs(tf.math.reduce_mean(signal, axis=1)) / self._snr  # calculate standard deviation for each signal
+            #stdd = tf.repeat(tf.expand_dims(stdd, 1), 11, axis=1)
+
+            # Use the std-dev as observed from the real normalised data
+            stdd = tf.random.uniform((signal.shape.as_list()[0],1), 0.07, 0.09)
```

Comment:
Could this be automated/loaded from preprocessing?

Reply:
This is an old version - the new noise generation code is different